docs/source/configuration/apollo-telemetry.mdx

```diff
@@ -304,6 +304,16 @@ You can configure whether the Apollo Router reports GraphQL error information to
 - To prevent your router from reporting error information at all, you can set the `send` option to `false`.
 - To include all error details in your router's reports to GraphOS, you can set the `redact` option to `false`.
 
+<blockquote>
+
+**Your subgraph libraries must support federated tracing (also known as FTV1 tracing) to provide errors to GraphOS.**
+
+- To confirm support, check the `FEDERATED TRACING` entry for your library on [this page](/federation/building-supergraphs/supported-subgraphs).
```

Comment:
```suggestion
To confirm support:
- Check the `FEDERATED TRACING` entry for your library on [this page](/federation/building-supergraphs/supported-subgraphs).
```

Reply:
Yep, this was taken verbatim from above section: https://www.apollographql.com/docs/router/configuration/apollo-telemetry#reporting-field-level-traces